src/server/api/utility.ts

```
@@ -0,0 +1,1311 @@
+
+  build(): Awaited<apiOk<T>> {
+    if (this.inputType !== "chain") {
+      // This will result in a TypeScript error
+      console.error("Cannot call build() on non-chain input");
+    }
+
+    const response: apiResponse<T> = {
+      status: this.status ?? apiResponseStatus.Inconsequential, // Default status if not set
+      type: this.type,
+      message: this._message,
+      data: this._data as T extends void | undefined | null ? undefined : T, // Cast to correct data type
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)